mm2src/coins/nft/storage/db_test_helpers.rs

```diff
@@ -158,6 +159,44 @@ fn nft_list() -> Vec<Nft> {
     };
 
     let nft2 = Nft {
+        common: NftCommon {
```

Comment:
Just for data consistency, since the NFTs in `nft_transfer_history` https://github.com/KomodoPlatform/komodo-defi-framework/blob/717e376447f9bcb76d972b2211ad22c83e2c287a/mm2src/coins/nft/storage/db_test_helpers.rs#L239 should be in `nft_list`